src/Factory.php

```diff
@@ -104,8 +110,12 @@ class Factory
         'single_import_per_statement'                 => false, // We like import grouping within the same namespace
         'single_line_throw'                           => false, // General soft line lengths cover this
         'single_trait_insert_per_statement'           => false, // We like grouping traits in one statement
+        'static_lambda'                               => true,
```

Comment:
Didn't we say we were going to turn this off? It's weird.

Reply:
Done...